app/src/main/java/com/duckduckgo/app/browser/print/PrintDocumentAdapterWrapper.kt

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright (c) 2022 DuckDuckGo
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.duckduckgo.app.browser.print
+
+import android.os.Bundle
+import android.os.CancellationSignal
+import android.os.ParcelFileDescriptor
+import android.print.PageRange
+import android.print.PrintAttributes
+import android.print.PrintDocumentAdapter
+
+class PrintDocumentAdapterWrapper(private val delegate: PrintDocumentAdapter) : PrintDocumentAdapter() {
```

Comment:
This class is not needed if we don't want to notify the activity when the printing finishes.

Reply:
True! I'll remove it 👍